mmcv/parallel/data_parallel.py

```diff
@@ -8,3 +9,15 @@ class MMDataParallel(DataParallel):
 
     def scatter(self, inputs, kwargs, device_ids):
         return scatter_kwargs(inputs, kwargs, device_ids, dim=self.dim)
+
+    def train_step(self, *inputs, **kwargs):
+        inputs, kwargs = self.scatter(inputs, kwargs,
+                                      [torch.cuda.current_device()])
+        output = self.module.train_step(*inputs[0], **kwargs[0])
+        return output
```

Comment:
Is the implementation correct?

Reply:
Ops, it is wrong. The latest codes have rewritten it.

Reply:
It seems still incorrect.